core/src/main/resources/hudson/triggers/SCMTrigger/BuildAction/index.jelly

```diff
@@ -30,15 +30,15 @@ THE SOFTWARE.
   <l:layout title="${it.run.parent.displayName} #${it.run.number} ${%Polling Log}">
     <st:include it="${it.run}" page="sidepanel.jelly" />
     <l:main-panel>
-      <h1>${%Polling Log}</h1>
       <l:rightspace>
+        <h1>${%Polling Log}</h1>
```

Comment:
That looks quite out of place
![](https://i.imgur.com/Gdthqvd.png)

I'd vouch for removing it from the rightspace at all and integrating it into the sidebar, like the "Console Output" -> "View as plain text" way e.g.:
![](https://i.imgur.com/ZyGOSxy.png)